python_package/madflow/makefile_template.py

```diff
@@ -0,0 +1,185 @@
+makefileName = "makefile"
+cppCompiler = "g++"
+cppVersion = "c++14"
+cudaPath = ""
```

Comment:
```suggestion
MAKEFILE = "makefile"
CPPCOMPILER = "g++"
CPPVERSION = "c++14"
```
As a convention constant variables are usually uppercase.